airgun/widgets.py

```diff
@@ -635,7 +635,7 @@ class Search(Widget):
     )
     search_field = TextInput(
         locator=(
-            ".//input[@id='search' or contains(@placeholder, 'Filter') or "
+            ".//input[@id='search' or contains(@placeholder, 'Search') or "
             "@ng-model='table.searchTerm' or contains(@ng-model, 'Filter') or "
```

Comment:
@shweta83 this change is not required, see line number 640. It already covers it(`contains(@placeholder, 'Search')`).

Reply:
Yes. I had old airgun so It was wasn't there. I undo that.